fe/fe-core/src/main/java/com/starrocks/qe/SessionVariable.java

```diff
@@ -1409,8 +1409,8 @@ public static MaterializedViewRewriteMode parse(String str) {
     @VariableMgr.VarAttr(name = NEW_PLANER_AGG_STAGE)
     private int newPlannerAggStage = SessionVariableConstants.AggregationStage.AUTO.ordinal();
 
-    @VariableMgr.VarAttr(name = TRANSMISSION_COMPRESSION_TYPE)
-    private String transmissionCompressionType = "NO_COMPRESSION";
+    @VariableMgr.VarAttr(name = TRANSMISSION_COMPRESSION_TYPE) 
+    private String transmissionCompressionType = "AUTO";
```

Comment:
will it be compatibility issue if FE upgraded to the newer version while the BE still stays in old version, especially during the upgrade.

Reply:
yes. it's a session variable, so the behavior on BE is totally driven by FE

Reply:
that means during the upgrade, FE sends to BE with "AUTO" but BE doesn't recognize and fail the request? or BE just fallback to the no_compression behavior?

Reply:
the best practice is upgrading the BE first

Reply:
isn't wise to count on the "best practice" which doesn't exist on any doc.

Reply:
well, actually it's a common practice of StarRocks, otherwise it would be over-complicated to add some new stuff in the codebase. the common practice is to use a session variable to control the behavior, so before upgrading the FE this new feature will not be enabled.